install.sh

```diff
@@ -661,6 +673,11 @@ rm -rf /var/lib/rancher/k3s
 rm -rf /var/lib/kubelet
 rm -f ${BIN_DIR}/k3s
 rm -f ${KILLALL_K3S_SH}
+
+if type yum >/dev/null 2>&1; then
+    yum remove -y k3s-selinux
+    rm -f /etc/yum.repos.d/rancher-k3s-common*.repo
```

Comment:
Does this throw an error if there isn't a repo file?

Reply:
Neither the `yum remove -y k3s-selinux` or the `rm -f` return errors even when there are no files/RPMs that match for removal, i.e.
```
[root@ck-centos7-0 ~]# ls
anaconda-ks.cfg
[root@ck-centos7-0 ~]# echo $?
0
[root@ck-centos7-0 ~]# rm -f lol
[root@ck-centos7-0 ~]# echo $?
0
[root@ck-centos7-0 ~]# yum remove -y k3s-selinux
Loaded plugins: fastestmirror
No Match for argument: k3s-selinux
No Packages marked for removal
[root@ck-centos7-0 ~]# echo $?
0
[root@ck-centos7-0 ~]#
```